nba/data_wrangling.py

```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+import numpy as np
+from pathlib import Path
+import sqlite3 as sql
+
+# Set the data path (adjust if necessary)
+DATA_PATH = Path.cwd() / "data" / "nba.sqlite"
+
+
+# Data loading function for insights 1 & 2
+def load_data(table_name: str) -> pd.DataFrame:
+    """Load data from the specified SQLite table"""
+    con = sql.connect(DATA_PATH)  # connect to the database
+    query = """
+    SELECT *
+    FROM draft_combine_stats
+    """
+    df = pd.read_sql(query, con)
+    con.close()
+    return df
+
+
+# Data loading function for insight 3
+def load_data_draft(table_name: str) -> pd.DataFrame:
+    """Load data from the specified SQLite table"""
+    con = sql.connect(DATA_PATH)  # connect to the database
+    query = """
+    SELECT *
+    FROM draft_history
+    """
+    df_draft = pd.read_sql(query, con)
+    con.close()
+    return df_draft
+
+
+# Data loading function for insight 4
+def load_data_team(table_name: str) -> pd.DataFrame:
+    """Load data from the specified SQLite table"""
+    con = sql.connect(DATA_PATH)  # connect to the database
+    query = """
+    SELECT *
+    FROM team_history
+    """
+    df_team = pd.read_sql(query, con)
+    con.close()
+    return df_team
+
+
+# Data wrangling functions
+def drop_empty_str_row(df: pd.DataFrame, col: str) -> pd.DataFrame:

```

Comment:
Ideally we would want to keep the docstring, which i think is in the original notebook?